test/run_test.vim

```diff
@@ -119,6 +119,8 @@ endfunction
 " Syntax test
 "-----------------------------------------------------------------------
 function! RunTestSyntax()
+    let g:verilog_syntax_fold_lst=""
+    colorscheme default
```

Comment:
This is only so I can run the test locally. My colourscheme (base16) defines extra highlight groups which causes the the HTML output to differ from the checked in HTML. Because the instance of vim that travis runs doesn't have these groups, its HTML output is 'simpler'.

I haven't explained this too well but If you want to go into it further then it's probably best to open an issue on this.